packages/framework/fluid-static/api-report/fluid-static.alpha.api.md

```diff
@@ -16,7 +16,7 @@ export interface ContainerSchema {
     readonly initialObjects: Record<string, SharedObjectKind>;
 }
 
-// @alpha @legacy
+// @beta @legacy
```

Comment:
These APIs should not be included in this rollup/report. Legacy APIs should not appear in non-legacy rollups/reports.

I think you've done most of the work required to clean this up. I think if you update the "report" API-Extractor config to point to the `alpha.d.ts` file, these should go away.

Let me know if that doesn't work or causes other issues. Happy to look at this with you :)

Reply:
Kk. Seems like this update will automatically generate after clean-up and re-build as well.

Reply:
The `legacy.alpha.api.md` report is being generated from an entry point `legacy.d.ts` that includes @beta @legacy APIs, but the report variant filtering should exclude anything that's not at the @alpha level. It generates both public and alpha report variants from the same entry point and not properly filtering @beta @legacy APIs out of the alpha variant. Should I separate legacy report generation?

Reply:
The variant filtering of at the "alpha" level instructs API-Extractor to include anything that is `@alpha` _or higher_, so it is expected that `@beta` and `@public` items appear. But since we're pointing it at a `.d.ts` file that has already omitted `@alpha` members, the result is no `@alpha` members in the report.

I think what you want to do is change the configured report variant to "beta" instead of "alpha". That will rename the report to match the contents we actually want in the report.

And if you do that, then you no longer need to add the alpha rollups / exports, which would let you revert some of the changes in your package.json file.